Sources/Submissions/Configs/TagTemplatePaths.swift

```diff
@@ -6,20 +6,25 @@ public struct TagTemplatePaths {
     public let emailField: String
     /// Path to template for input of type "password"
     public let passwordField: String
+    /// Path to template for textarea
+    public let textareaField: String
 
     /// Create a new TagTemplatePaths configuration value.
     ///
     /// - Parameters:
     ///   - textField: path to template for input of type "text"
     ///   - emailField: path to template for input of type "email"
     ///   - passwordField: path to template for input of type "password"
+    ///   - textareaField: path to template for textarea
     public init(
         textField: String = "Submissions/Fields/text-input",
         emailField: String = "Submissions/Fields/email-input",
```

Comment:
if you move this one up it's nice and alphabetized again :) see also properties and assignments within `init`.